lucene/directory-provider/src/test/java/org/infinispan/hibernate/search/ClusterTestHelper.java

```diff
@@ -23,6 +23,56 @@
  */
 public final class ClusterTestHelper {
 
+   public enum ExclusiveIndexUse {
+      EXCLUSIVE {
+         void apply(FullTextSessionBuilder node) {
+            node.setProperty(key, "true");
+         }
+      },
+      SHARED {
+         void apply(FullTextSessionBuilder node) {
+            node.setProperty(key, "false");
+         }
+      };
+      private static final String key = "hibernate.search.default." + org.hibernate.search.cfg.Environment.EXCLUSIVE_INDEX_USE;
+      abstract void apply(FullTextSessionBuilder node);
+   }
+
+   public enum IndexManagerType {
+      TRADITIONAL_DIRECTORYPROVIDER {
+         void apply(FullTextSessionBuilder node) {
+            node.setProperty("hibernate.search.default.directory_provider", "infinispan");
+         }
+      },
+      DEDICATED_INDEXMANAGER {
+         void apply(FullTextSessionBuilder node) {
+            node.setProperty("hibernate.search.default.indexmanager", "infinispan");
+         }
+      };
+      abstract void apply(FullTextSessionBuilder node);
+   }
+
+   public enum IndexingFlushMode {
+      ASYNC_PERIODIC {
+         void apply(FullTextSessionBuilder node) {
+            node.setProperty("hibernate.search.default.index_flush_interval", "1");
+            node.setProperty("hibernate.search.default.worker.execution", "async");
+         }
+      },
+      ASYNC { //Deprecated?
```

Comment:
I'll clarify the comment.

I simply mean it a as reminder to ourselves that using "async" without using your periodic commit is probably not a configuration which makes much sense.